packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/general-information/general-information.controller.js

```diff
@@ -27,9 +28,27 @@ export default class {
         vScope: this.currentService.vScopeUrl,
         webInterface: this.currentService.webInterfaceUrl,
       },
+      updateAvailable:
+        this.isMinorSolutionUpdateAvailable() ||
+        this.isMajorSolutionUpdateAvailable(),
+      version: this.currentService.version,
     };
   }
 
+  isMinorSolutionUpdateAvailable() {
+    const lastMinor = get(this.currentService, 'version.lastMinor');
+    return (
+      lastMinor &&
+      lastMinor.major === this.currentService.version.major &&
+      lastMinor.build !== this.currentService.version.build
+    );
+  }
+
+  isMajorSolutionUpdateAvailable() {
```

Comment:
do you think its good idea to move this to a class?

Reply:
This is done. Thanks :)